python/ray/serve/_private/http_proxy.py

```diff
@@ -909,6 +879,10 @@ def __init__(
         configure_component_logger(
             component_name="http_proxy", component_id=node_ip_address
         )
+        logger.info(
+            f"Start the http proxy actor {ray.get_runtime_context().get_actor_id()} "
+            f"on node {node_id}"
```

Comment:
proper punctuation & grammar please
```suggestion
            f"Proxy actor {ray.get_runtime_context().get_actor_id()} "
            f"starting on on node {node_id}."
```